Modules/xSharePoint/DSCResources/MSFT_xSPAppCatalog/MSFT_xSPAppCatalog.psm1

```diff
@@ -0,0 +1,79 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $AppCatalogUrl ,
```

Comment:
Note on this: I thought you Wanted the values separate.
We could have a single value, that would reduce code as I am parsing and validating the relative URL as well


Sent via iPhone while out and about


> On 22 Dec 2015, at 11:21 PM, Brian Farnhill <notifications@github.com> wrote:
> 
> In Modules/xSharePoint/DSCResources/MSFT_xSPAppCatalog/MSFT_xSPAppCatalog.psm1:
> 
> > @@ -0,0 +1,79 @@
> > +function Get-TargetResource
> > +{
> > +    [CmdletBinding()]
> > +    [OutputType([System.Collections.Hashtable])]
> > +    param
> > +    (
> > +        [parameter(Mandatory = $true)] [System.String] $AppCatalogUrl ,
> Do we need app catalog URL and web app url here? Can't we just look up the web app that the site collection is in? Note that the update powershell method we use only takes the site in, not the web app URL as well.
> 
> —
> Reply to this email directly or view it on GitHub.
>